carah_app/lib/model/lightContent.dart

```diff
@@ -0,0 +1,20 @@
+
+import 'content.dart';
+
+enum ContentType {
+  none,
+  article,
+  question
+}
+
+class LightContent extends Content{
+  late ContentType contentType;
+
+  LightContent({required String uuid, required String title, required String content, required String category}) : super(uuid: uuid, title: title, content: content, category: category);
```

Comment:
Mini Anmerkung: wenn du den Konstruktor aufrufst, dann übergibst du immer content: "". Dafür könntest du im Konstruktor auch direkt einen Default-Wert übergeben, dann spart man sich das jedes mal :)
```suggestion
  LightContent({required String uuid, required String title, String content = "", required String category}) : super(uuid: uuid, title: title, content: content, category: category);
```